src/Xrefcheck/Orphans.hs

```diff
@@ -0,0 +1,37 @@
+{- SPDX-FileCopyrightText: 2021 Serokell <https://serokell.io>
+ -
+ - SPDX-License-Identifier: MPL-2.0
+ -}
+
+{-# OPTIONS_GHC -fno-warn-orphans #-}
+
+-- | Orphan instances for types from other packages
+
+module Xrefcheck.Orphans () where
+
+import qualified Data.ByteString as BS
+
+import Fmt (Buildable (..), (+|), (|+))
+import Network.FTP.Client
+  (FTPException (..), FTPMessage (..), FTPResponse (..), ResponseStatus (..))
+
+instance Buildable ResponseStatus where
+  build = show
+
+instance Buildable FTPMessage where
+  build message = build $ decodeUtf8 @Text (
+    case message of
+      SingleLine s -> s
+      MultiLine ss -> BS.concat ss
+    )
+
+instance Buildable FTPResponse where
+  build FTPResponse{..} =  frStatus |+ " (" +| frCode |+ ") " +| frMessage |+ ""
```

Comment:
This is probably too much info to display in one single line. Try formatting it across multiple lines (Hint: `unlinesF` will make this easier)

Reply:
not sure I've got your point..
do you mean this?
```
instance Buildable FTPResponse where
  build FTPResponse{..} = unlinesF
    [ mconcat [build frStatus, " (", build frCode, "):"]
    , build frMessage
    ]
```

Reply:
Yup, I meant that using `unlinesF` is better than concatenating strings and manually adding `\n` in between them.

Btw, for formatting a single line, I think `frStatus |+ " (" +| frCode |+ "):"` reads better than `mconcat [build frStatus, " (", build frCode, "):"]`